tsdb/engine/tsm1/engine.go

```diff
@@ -2251,7 +2265,7 @@ func (e *Engine) compactLoPriorityLevel(grp CompactionGroup, level int, fast boo
 
 // compactFull kicks off full and optimize compactions using the lo priority policy. It returns
 // the plans that were not able to be started.
-func (e *Engine) compactFull(grp CompactionGroup, wg *sync.WaitGroup) bool {
+func (e *Engine) compactFull(grp CompactionGroup, wg *sync.WaitGroup, pointsPerBlock int) bool {
```

Comment:
I like passing `pointsPerBlock` in, but I'd like to see `pointsPerBlock` after `grp`. That way the compaction parameters are kept together.